app/views/organization_pets/_form.html.erb

```diff
@@ -6,7 +6,7 @@
                             value: current_user.staff_account.organization_id %>
 
       <div class='form-group'>
-        <%= form.label :name, class: 'mt-3' %>
+        <%= form.label :name, 'Name*', class: 'mt-3' %>
```

Comment:
Could you check to see if adding `required: true` adds the asterisk for us? I normally see the asterisk added using that approach.

Reply:
Hi @edwinthinks ! I tried to use `required: true`, but it wasn't applying the asterisk. I followed [this file](https://github.com/rubyforgood/pet-rescue/blob/main/app/views/contacts/new.html.erb#L32-L35) 🤔

Reply:
Hmm I suspect it is because this is using form_with and not `form_for` could you see if that makes a difference?